src/main/webapp/WEB-INF/jsp/login.jsp

```diff
@@ -249,8 +249,8 @@
                 </div>
 
                 <div class="login-list">
-                    <input class="button" type="submit" name="save"
-                           onclick="document.getElementById('username').value = document.getElementById('username').value.trim()"
+                    <input class="button" type="submit" name="save" data-prevent-double-click="true"
+                           onclick="document.getElementById('username').value = document.getElementById('username').value.trim(); this.disabled = true; document.getElementsByName('loginForm')[0].submit()"
```

Comment:
We do it like this in a couple of other places, but whatever works is fine:
https://github.com/hmcts/idam-web-public/blob/aea0193a23323ecb7d3a3e17e1d0314942838575/src/main/webapp/WEB-INF/jsp/resetpassword.jsp#L65

Reply:
I tried this but it seems it didn't look like it submitted during the disabled duration, maybe I should try again 🤔